nrf_rpc/nrf_rpc.c

```diff
@@ -465,6 +467,27 @@ static int transport_init(nrf_rpc_tr_receive_handler_t receive_cb)
 	return err;
 }
 
+static int transport_init_all(nrf_rpc_tr_receive_handler_t receive_cb)
+{
+	void *iter;
+	const struct nrf_rpc_group *group;
+
+	for (NRF_RPC_AUTO_ARR_FOR(iter, group, &nrf_rpc_groups_array,
+				 const struct nrf_rpc_group)) {
+
+		transport_init_single(receive_cb, group);
+	}
+
+	/* Group initialization errors are not propagated to the caller. */
+	return groups_init_event_wait();
+}
+
+static void default_err_handler(const struct nrf_rpc_err_report *report)
+{
+	NRF_RPC_ERR("nRF RPC error %d ocurred. See nRF RPC logs for more details", report->code);
+	k_oops();
```

Comment:
This should be OS-independent code. k_oops() is a Zephyr function. Please pass it through OS porting layer.

Reply:
Regarding this, do you refer to this OS porting layer do you refer to the nrf_rpc one?
So what you are proposing is to add another function here:
https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/nrf_rpc/nrf_rpc_os.c

Which handles the error case?

Reply:
Do you have an opinion on this Pawel? I can  do what I am describing here if you have the same understanding as me